tests/test_readMcool.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
+#
+# Testing for the helper function readMcool in hicrep/utils
+#
+import pytest
+from hicrep.utils import readMcool
+
+def testReadMcool():
```

Comment:
I think most of the tests here are covered already by the cooler package...

Reply:
Removed some of the tests - I left the ones that checked bin size just as a sanity check that we are reading in at the right resolution. Feel free to axe them too if you think it's best.